coalib/bearlib/languages/definitions/Haskell.py

```diff
@@ -0,0 +1,15 @@
+from coalib.bearlib.languages.Language import Language
+
+
+@Language
+class Haskell:
+    extensions = '.hs',
+    versions = 1.0, 1.1, 1.2, 1.3, 1.4
```

Comment:
Haskell versioning is quite unique. The major version `1.x` is before Haskell 98 released.

I would list them as `1.0, 1.1, 1.2, 1.3, 1.4, 98, 2010`. However, we need to clarify this.